README.md

```diff
@@ -13,39 +13,88 @@ This is a part of the article: [Testing Angular faster with Jest](https://www.xf
 
 _Note: This preset does not support AngularJS (1.x). If you want to set up Jest with AngularJS, please see [this blog post](https://medium.com/aya-experience/testing-an-angularjs-app-with-jest-3029a613251)._
```

Comment:
I would suggest to remove this part on the next major version (and the link to the article as well, as they both are pretty outdated), if thymikee is ok with it.

Reply:
It's about time, let's do it 👍

Reply:
ok I will update this in #562

Reply:
added in https://github.com/thymikee/jest-preset-angular/pull/562/commits/65bedff31f5a396192c15ad916ae7d9bd2755fdd